content/techniques/mongo.md

```diff
@@ -70,6 +70,24 @@ Alternatively, the `@Prop()` decorator accepts an options object argument ([read
 name: string;
 ```
 
+In case if you want to specify relation to another model, later for pupulating, you can use `@Prop()` decorator as well. For example, if `Cat` has `Owner` which is stored in different collection called `owners`, property should have type and ref. For example:
```

Comment:
`In case you` not `In case if you`
`populating` not `pupulating`
`, the property` not `, property`

Reply:
> `In case you` not `In case if you`
> `populating` not `pupulating`
> `, the property` not `, property`

Fixes are in ea37bac9cc283eae1fcd8f5c317df9129be8d816